### ISSUE TITLE
Save parameters widget input values and load them from a dictionary

### ISSUE THREAD

**Issue by YooSunYoung** [scipp/ess#108]
We want to let users save parameters and auto-complete the parameter inputs from it.

**Comment by jl-wynen** [comment]
This is related to https://github.com/scipp/sciline/issues/92. So ideally, the solution here can also be used with the JSON repr of Sciline. Even if it is not a complete solution to that issue.

**Comment by SimonHeybrock** [comment]
I thought it should be distinct: Saving workflow parameters is different from storing the widget state. As a simple example, the workflow input parameters might be some bin-edges (an "arbitrary" variable), while the widget state gives start, stop, nbin, and scale (lin or log).

But maybe there is a better abstraction?

**Comment by YooSunYoung** [comment]
Yeah, it's not really possible to fill widget inputs from a parameter value from the graph, i.e. scipp.Variable.
Maybe the title was misleading. I'll update it.

**Comment by YooSunYoung** [comment]
There are some widgets that needs explicit handling.

TODO:
- [ ] ~Dropdown widget~ Find `set_fields` or `value` setter.  - scipp/essreduce#149 
- [ ] Optional widget (wrapper) - scipp/essreduce#148 
- [ ] Switchable widget (wrapper)
